migration/sql-files/006-external-provider.sql

```diff
@@ -0,0 +1,22 @@
+
+-- token storage
+
+CREATE TABLE external_providers (
+    id uuid primary key DEFAULT uuid_generate_v4() NOT NULL,
+    created_at timestamp with time zone,
+    updated_at timestamp with time zone,
+    deleted_at timestamp with time zone,
+    type text,
+    url text
+);
+
+CREATE TABLE external_provider_tokens (
+    id uuid primary key DEFAULT uuid_generate_v4() NOT NULL,
+    created_at timestamp with time zone,
+    updated_at timestamp with time zone,
+    deleted_at timestamp with time zone,
+    external_provider_id uuid  REFERENCES external_providers (id),
+    identity_id uuid REFERENCES identities(id),
```

Comment:
We need an index by identity id.

Reply:
added a couple of them .. for identity_id and provider_id
https://github.com/fabric8-services/fabric8-auth/pull/131/files#diff-65b55834ac486723e3e9bf843295d903R15